Toolchain/BuildClang.sh

```diff
@@ -36,7 +36,6 @@ elif [ "$SYSTEM_NAME" = "Darwin" ]; then
     MD5SUM="md5 -q"
     REALPATH="grealpath"  # GNU coreutils
     INSTALL="ginstall"    # GNU coreutils
-    SED="gsed"            # GNU sed
```

Comment:
if we don't need sed at all, it should be removed on line 23 as well.

Reply:
We do need sed, but the sed that comes with macOS suffices for the commands in this file (line 81 and 262).

Reply:
Or do you mean we should hardcode it to `sed` on those lines, and remove the variable on line 23?

Reply:
Shouldn't it just be `SED="sed"`?

Reply:
> Shouldn't it just be `SED="sed"`?

That is already the value after line 23.